src/read.c

```diff
@@ -4153,7 +4153,8 @@ static avifResult avifParse(avifDecoder * decoder)
         }
 #endif
 
-        if (!isFtyp && (isNonSkippableVariableLengthBox || !memcmp(header.type, "free", 4) || !memcmp(header.type, "mdat", 4))) {
+        if (!isFtyp && (isNonSkippableVariableLengthBox || !memcmp(header.type, "free", 4) || !memcmp(header.type, "skip", 4) ||
+                        !memcmp(header.type, "mdat", 4))) {
```

Comment:
For AVIF, it may be fine to require the FileTypeBox be the first box in the file. The only example given in ISO/IEC 14496-12 that may go before the FileTypeBox is a file signature, but AVIF doesn't have a file signature.